src/patternfly/components/DescriptionList/description-list.scss

```
@@ -107,17 +114,29 @@ $pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "md",
       // stylelint-enable
     }
   }
+
+  &.pf-m-display-lg {
+    .pf-c-description-list__description {
+      font-size: var(--pf-c-description-list--m-display-lg__description--FontSize);
+    }
```

This works but you could add 

```
.pf-c-description-list__description {
  font-size: var(--pf-c-description-list__description--FontSize, inherit);
}
```

Then update this block to 

```suggestion
  --pf-c-description-list__description--FontSize: var(--pf-c-description-list--m-display-lg__description--FontSize);
```